service/metrics.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 LY Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"github.com/kpango/glg"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Metrics interface {
+	GetOriginLatency() prometheus.Histogram
+}
```

Comment:
remove prometheus in interface
```suggestion
const (
    HTTP_ORIGIN_LATENCY = "http_origin_latency_in_seconds"
)

type Metrics interface {
	Observe(name string, labelValues []string, value) error
}
```

Reply:
@WindzCUHK fixedin https://github.com/AthenZ/authorization-proxy/pull/30/commits/1a0bc8cef3624f605e59ddad85deec897964644b